### PR TITLE
Issue list filters

### PR DESCRIPTION
**Pull request by bellhwi**

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-21)

File: features/issues/components/issue-list/issue-filter.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  const levelSelectOptions = ["All level", "Error", "Warning", "Info"];
+  const issues = useSelector((state: any) => state.issues);
+
+  return (
+    <Container>
+      <StyledButton icon={ButtonIcon.leading}>
+        Resolve selected issues
+      </StyledButton>
+      <FilterContainer>
+        <div style={{ width: "160px", height: "44px", marginRight: "16px" }}>
```
Comment (jkettmann) [inline features/issues/components/issue-list/issue-filter.tsx:35]: I wouldn't mix styled-components and inline style if possible

File: features/issues/components/issue-list/issue-filter.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            state={statusSelectOpen ? SelectState.open : SelectState.filled}
+            options={stateSelectOptions}
+            defaultOption={stateSelectOptions}
+            selectedIndex={0}
+            onClick={() => {
+              if (statusSelectOpen) {
+                setStatusSelectOpen(false);
+              } else {
+                setStatusSelectOpen(true);
+              }
```
Comment (jkettmann) [inline features/issues/components/issue-list/issue-filter.tsx:46]:
```
() => setStatusSelectOpen(!statusSelectOpen)
```

File: features/issues/components/issue-list/issue-filter.tsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+`;
+
+export const IssueFilter = () => {
+  const [statusSelectOpen, setStatusSelectOpen] = useState(false);
+  const [levelSelectOpen, setLevelSelectOpen] = useState(false);
+  const [currentState, setCurrentState] = useState("All status");
+  const [currentLevel, setCurrentLevel] = useState("All level");
+  const stateSelectOptions = ["All status", "Unresolved", "Resolved"];
+  const levelSelectOptions = ["All level", "Error", "Warning", "Info"];
+  const issues = useSelector((state: any) => state.issues);
```
Comment (jkettmann) [inline features/issues/components/issue-list/issue-filter.tsx:27]: No need for this, you could simply use the `useGetIssues` hook here as well (same as in the `issue-list` component). That makes Redux overall obsolete

**Pull request merged**